src/compiler.cairo

```diff
@@ -21,6 +22,7 @@ pub impl CompilerTraitImpl of CompilerTrait {
         opcodes.insert('OP_0', Opcode::OP_0);
         opcodes.insert('OP_1', Opcode::OP_1);
         opcodes.insert('OP_ADD', Opcode::OP_ADD);
+        opcodes.insert('OP_DEPTH', Opcode::OP_DEPTH);
```

Comment:
I'd like to keep these in order based on byte code; otherwise, it will be harder to maintain later. Thank you!
( So in this example, it should go before OP_ADD )

Reply:
@b-j-roberts done